src/plugins/pluginManager.ts

```diff
@@ -148,12 +148,17 @@ const getPlugin = (pluginId: string) => {
   return plugins[pluginId];
 };
 
+const unloadPlugin = (pluginId: string) => {
+  plugins[pluginId] = undefined;
+};
```

Comment:
option to unload plugin so you dont need to restart app to apply plugin settings